source/country.js

```diff
@@ -1,7 +1,6 @@
+Country = Space.domain.ValueObject.extend('Country', {
 
-Country = Space.messaging.Serializable.extend('Country', {
-
-  Constructor: function(country) {
+  Constructor(country) {
 
     // Allow to provide another instance of Country as param
     if(country instanceof Country) {
```

Comment:
I suggest removing this:
`country = (country && country.code) ? country.code : country;`

IMO we should strictly accept instances of the same VO or plain objects. Or other VOs in some cases but not plain classes. It becomes very messy to handle all possible construction options in some cases.